src/playbooks/_database_connection/metadata.obsah.yml

```diff
@@ -0,0 +1,14 @@
+---
+variables:
+  database_host:
+    help: Host of the PostgreSQL database.
+  database_port:
+    help: Port of the PostgreSQL database.
+  foreman_database_host:
+    help: Hostname of the PostgreSQL server.
```

Comment:
```suggestion
```

This doesn't need to be defined here, it will be created internally by `vars/database.yml`